File: benchmarks/xlang/src/fib.go
package main
import ("fmt"; "time")
func fib(n int64) int64 { if n < 2 { return n }; return fib(n-1) + fib(n-2) }
func main() {
    t0 := time.Now()
    r := fib(32)
    fmt.Printf("CHECK %d\nMS %d\n", r, time.Since(t0).Milliseconds())
}

File: benchmarks/xlang/src/fib.cpp
#include <cstdio>
#include <chrono>
long fib(long n) { return n < 2 ? n : fib(n - 1) + fib(n - 2); }
int main() {
    auto t0 = std::chrono::steady_clock::now();
    long r = fib(32);
    auto ms = std::chrono::duration_cast<std::chrono::milliseconds>(
        std::chrono::steady_clock::now() - t0).count();
    printf("CHECK %ld\nMS %lld\n", r, (long long)ms);
}
